packages/pipeline/src/scripts/pull_ohlcv_cryptocompare.ts

```diff
@@ -0,0 +1,92 @@
+// tslint:disable:no-console
+import { Connection, ConnectionOptions, createConnection, Repository } from 'typeorm';
+
+import { CryptoCompareOHLCVSource } from '../data_sources/ohlcv_external/crypto_compare';
+import { OHLCVExternal } from '../entities';
+import * as ormConfig from '../ormconfig';
+import { OHLCVMetadata, parseRecords } from '../parsers/ohlcv_external/crypto_compare';
+import { handleError } from '../utils';
+import { fetchOHLCVTradingPairs, TradingPair } from '../utils/get_ohlcv_trading_pairs';
+
+const SOURCE_NAME = 'CryptoCompare';
+
+// convert these to env variables
+const MAX_CONCURRENT_REQUESTS = 18;
+const EARLIEST_BACKFILL_TIME = new Date('2010-09-01').getTime(); // the time when BTC/USD info starts appearing on Crypto Compare
+
+let connection: Connection;
+
+(async () => {
+    connection = await createConnection(ormConfig as ConnectionOptions);
+    const repository = connection.getRepository(OHLCVExternal);
+    const source = new CryptoCompareOHLCVSource(MAX_CONCURRENT_REQUESTS);
+
+    const tradingPairs = await fetchOHLCVTradingPairs(connection, SOURCE_NAME, EARLIEST_BACKFILL_TIME);
```

Comment:
Always suffix methods that are async with `Async`, e.g `fetchOHLCVTradingPairsAsync`